src/implementations/react/src/elements/components/Typography.js

```diff
@@ -0,0 +1,120 @@
+// 'h1', 'h2', 'h3', 'sub1', 'sub2', 'body', 'bold', 'disabled', 'caption'
+import React from 'react';
+import Typography from '../../adapters/TypographyAdapter'
+import * as PropTypes from 'prop-types'
+
+class H1Component extends React.Component {
+  render() {
+    return (<Typography type="h1" text={this.props.children} />)
+  }
+}
+
+class H2Component extends React.Component {
+  render() {
+    return (<Typography type="h2" text={this.props.children} />)
+  }
+}
```

Comment:
It's good practice in React to create "pure" components (AKA functions rather than classes) unless you need a constructor to hold state or to do all the crazy stuff our adapters do.